controllers/automation/policyautomation_controller.go

```diff
@@ -89,6 +92,150 @@ func (r *PolicyAutomationReconciler) setOwnerReferences(
 	return nil
 }
 
+// getTargetListMap will convert slice targetList to map for search efficiency
+func getTargetListMap(targetList []string) map[string]bool {
+	targetListMap := map[string]bool{}
+	for _, target := range targetList {
+		targetListMap[target] = true
+	}
+
+	return targetListMap
+}
+
+// getClusterName will get the hub cluster name information
+func getClusterName() string {
+	// Get a config to talk to the apiserver
+	cfg := config.GetConfigOrDie()
```

Comment:
In theory this should never fail, but I don't think a panic is necessary here. I think returning an empty string if the config can't be retrieved would be preferable.